extensions/amp-sidebar/0.1/test/validator-amp-sidebar2.out

```diff
@@ -0,0 +1,2 @@
+FAIL
+amp-sidebar/0.1/test/validator-amp-sidebar2.html:40:6 Tag 'amp-accordion > section' must have 2 child tags - saw 1 child tags. [AMP_TAG_PROBLEM]
```

Comment:
Github says this file does not end with a newline. Sorry it could be that our test setup doesn't complain about this but it's probably better to add it just in case.